pkgs/development/python-modules/pki-core/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, lib, fetchurl, buildPythonPackage, sphinx, rpmextract, requests,
+  six }:
+
+buildPythonPackage rec {
+  name = "pki-core-${version}";
+  version = "10.10.5-6";
+
+  rpmFile = "${name}-7.fc25.src.rpm";
+
+  src = fetchurl {
+    url = "mirror://fedora/linux/releases/25/Everything/source/tree/Packages/p/${rpmFile}";
+    sha256 = "18kva88qqjfbkccaw1j889jfm95zk8a63xhy05ldj07yfqz38vsz";
+  };
+
+  patches = [
+    "pki-core-snapshot-1.patch"
+    "pki-core-snapshot-2.patch"
+    "pki-core-snapshot-3.patch"
+    "pki-core-snapshot-4.patch"
+    "pki-core-fedora-post-snapshot-1.patch"
+  ];
+
+  patchFlags = "-p1 -F 3";
+
+  nativeBuildInputs = [ rpmextract ];
+  buildInputs = [ sphinx ];
```

Comment:
Is sphinx run as part of the build process?

Reply:
Removed pki-core entirely, as it is not in use.